Dockerfile.build

```diff
@@ -0,0 +1,9 @@
+FROM gradle:5.4.1-jdk11
+LABEL maintainer="Rob March <rob.march@digirati.com>"
+
+USER root
```

Comment:
Why are we doing this?

Reply:
The `apt-get` commands were failing due to not having the appropriate permissions (e.g. https://jenkins.build.digirati.com/job/digirati-taxonomy-manager/job/PR-1/3/console ). The solutions to it that I saw online all suggested running the command as root but this seemed like a cleaner solution than putting `sudo`s for each of the `apt-get` commands

Reply:
We should switch back to the original user before we actually run the commands (e.g., switch to root, install packages, switch back, run build). It's relatively trivial to escape a container as root.

Reply:
Er, when I say _the commands_, I mean the build commands. So our `./gradlew build` or whatever.

Reply:
I prefer using `sudo` rather than switching to `root`, even it's only during the image build.

Reply:
`sudo` might not be available in the image. In fact, it shouldn't be.

Reply:
We can install it for the purposes of building the image, and if we use a multi-stage build we don't have to carry that dependency over to the final image.

Reply:
I'm not understanding why we would install an external package from inside the build to switch UIDs when Docker supports doing it for us.